### PR TITLE
[SIG-2775] update asc packages

### PR DESCRIPTION
**Pull request by aditudorache**
Thie PR updates the `asc` packages to 0.20.0 and solves a breaking change around the the site Logo. A SIA variant of the Amsterdam Logo is added to this project because the default logo provided by asc is not according to the design of SIA.

### CODE REVIEW

File: src/components/AmsterdamLogo/index.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { LogoShort } from '@datapunt/asc-assets';
+import styled, { css } from 'styled-components';
+import { breakpoint, themeSpacing, themeColor } from '@datapunt/asc-ui';
+
+export const srOnlyStyle = () => ({ srOnly }) =>
+  srOnly
```
Comment (janjaap) [inline src/components/AmsterdamLogo/index.js:7]: I would be enough to do
```jsx
({ srOnly }) => srOnly && css``;
```

File: src/components/AmsterdamLogo/index.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { LogoShort } from '@datapunt/asc-assets';
+import styled, { css } from 'styled-components';
+import { breakpoint, themeSpacing, themeColor } from '@datapunt/asc-ui';
+
+export const srOnlyStyle = () => ({ srOnly }) =>
```
Comment (janjaap) [inline src/components/AmsterdamLogo/index.js:6]: What is `srOnly` and why should there be a check for it when the prop `srOnly` is always true?

File: src/components/AmsterdamLogo/index.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+export const LogoTitleStyle = styled.span`
+  ${srOnlyStyle()}
+`;
+
+const AmsterdamLogoStyle = styled.a`
+  display: inline-block;
+  height: 30px;
+  width: 68px;
+  flex-shrink: 0;
```
Comment (janjaap) [inline src/components/AmsterdamLogo/index.js:34]: `flex-shrink` Doesn't have any effect when the element isn't a flex element. In this case it's `inline-block`, not `flex` or `inline-flex`.

File: internals/webpack/webpack.base.babel.js [hunk 4]
```diff
@@ -138,6 +139,10 @@ module.exports = options => ({
     modules: ['node_modules', 'src'],
     extensions: ['.js', '.jsx', '.react.js'],
     mainFields: ['browser', 'jsnext:main', 'main'],
+    alias: {
+      '@datapunt/asc-ui': path.resolve(__rootdir, 'node_modules/@datapunt/asc-ui/lib'),
+      '@datapunt/asc-assets': path.resolve(__rootdir, 'node_modules/@datapunt/asc-assets/lib'),
+    },
```
Comment (janjaap) [inline internals/webpack/webpack.base.babel.js:145]: Why wouldn't we make use of esModules? When I apply them, I'm getting a reduction in production bundle size. Not significant, but still a reduction.

File: src/components/AmsterdamLogo/index.js (hunk 1 above)
Comment (aditudorache) [reply src/components/AmsterdamLogo/index.js:7]: indeed

File: src/components/AmsterdamLogo/index.js (hunk 2 above)
Comment (aditudorache) [reply src/components/AmsterdamLogo/index.js:6]: The screen readers variant of the logo. I removed that.

File: src/components/AmsterdamLogo/index.js (hunk 3 above)
Comment (aditudorache) [reply src/components/AmsterdamLogo/index.js:34]: correct, I missed that,removed.

File: internals/webpack/webpack.base.babel.js (hunk 4 above)
Comment (aditudorache) [reply internals/webpack/webpack.base.babel.js:145]: I didn't know we don't support IE11 anymore.

**Pull request merged**